src/lib/date/relativeTime.ts

```diff
@@ -0,0 +1,48 @@
+import i18n from '@dhis2/d2-i18n'
+
+type TimeUnit = {
+    unit: Intl.RelativeTimeFormatUnit
+    ms: number
+}
+
+const units: TimeUnit[] = [
+    { unit: 'year', ms: 31536000000 },
+    { unit: 'month', ms: 2628000000 },
```

Comment:
I guess we'll have some weirdness around February / at leap years, but I don't think that's an issue worth fine tuning for here. It makes me wonder if that's the reason why moment.fromNow() rounds up though 🤔

Reply:
Hm... We're just calculating the actual value of time, though, so I guess the "worst" that could happens is that you would get `11 months ago` instead of a `1 year ago`, though, right? And since we're just counting actual seconds, if you get an extra day - that would be counted in the timestamp, and be correct? I haven't been dealing with dates and timezone as much, so there's probably something Im not thinking about. But yeah, I dont think needs to be that accurate.